### PR TITLE
First pass of refactoring Angular deps to use mod.name to avoid duplication

### PR DESCRIPTION
**Pull request by theefer**
Discovered this way of doing things in a sample Angular project and it feels nicer to me, using the module.name of imported models rather than treating imports purely as side-effects and duplicating the module name manually. Also stricter with each file being its own module, to make it easier to reason about.

Haven't converted the whole codebase yet, only things referred to by main.js (and fixed anything that broke).

### CODE REVIEW

**Comment by jamesgorrie** [comment]
Nice! That'll make it work a lot better with IDEs too (I can actually see where the dep comes from).
:+1:

File: kahuna/public/js/services/api/loader.js [hunk 1]
```diff
@@ -1,8 +1,11 @@
-import apiServices from '../api';
+import angular from 'angular';
+import {mediaApi} from './media-api';
 
-apiServices.factory('loaderApi',
-                    ['mediaApi',
-                     function(mediaApi) {
+export var loaderApi = angular.module('kahuna.services.api.loader', [
```
Comment (jamesgorrie) [inline kahuna/public/js/services/api/loader.js]: You could change these to `const`s

File: kahuna/public/js/crop/index.js [hunk 2]
```diff
@@ -2,13 +2,15 @@ import angular from 'angular';
...
+import '../directives/ui-crop-box';
 
 import cropTemplate from './view.html!text';
 
 
 export var crop = angular.module('kahuna.crop', [
     'kahuna.crop.controller',
-    'gr.topBar'
+    'gr.topBar',
+    'ui.cropBox'
```
Comment (akash1810) [inline kahuna/public/js/crop/index.js]: how/why was this module working before w/out this?

File: kahuna/public/js/services/api/loader.js (hunk 1 above)
Comment (theefer) [inline kahuna/public/js/services/api/loader.js]: I thought about that but stuck to `var` for consistency with the rest of the codebase. We can do a general replace throughout as a separate step.

I'm not sure what it means for an export to be `var` or `const`, and specifically what happens to the exported value if you reassign it later..

File: kahuna/public/js/crop/index.js (hunk 2 above)
Comment (theefer) [inline kahuna/public/js/crop/index.js]: It was imported in `main.js`, I moved it to be more local.

Comment (akash1810) [inline kahuna/public/js/crop/index.js]: Ah, must have missed that. :+1: for local imports!

**Pull request merged**